code/api/src/modules/product/query.test.js

```diff
@@ -0,0 +1,54 @@
+import request from 'supertest'
+import express from 'express'
+import schema from '../../setup/schema'
+import graphqlHTTP from 'express-graphql'
+
+describe('product query', () => {
+  let server;
+
+  beforeAll(() => {
+    server = express();
+
+    server.use(
+      "/", 
+      graphqlHTTP({
+        schema: schema,
+        graphiql: true
+      })
+    )
+  })
+
+  it("is true", () => {
+    expect(true).toBe(true)
+  })
+
+  it("returns products with isSurvey: false", async () => {
+    const response = await request(server)
+    .get('/')
+    .send({
+      query: `{productSurveyTrue(productSurvey: false){
+      name
+      id
+      isSurvey
+      }}`
+    })
+    .expect(200)
+    console.log(response.body)
+    expect(response.body.data.productSurveyTrue.length).toEqual(8)
```

Comment:
This test looks good, My only concern is that it is using the seeded db and won't be immediately apparent where these 8 values are coming from by looking at the test. Not really an issue for us now but in theory down the road it would be more difficult for another dev to figure out why this should return 8.

Reply:
True, I can try to fix that and test it another way

Reply:
If you want to maybe just add a comment for now that lets people know that the seeds are where the products are coming from, then later if we have time we can update the test itself. This way we can get it merged quicker for FE to be able to build off of.